pkg/build/webhook/github/github.go

```diff
@@ -53,8 +53,8 @@ func (p *WebHook) Extract(buildCfg *api.BuildConfig, secret, path string, req *h
 		return revision, envvars, dockerStrategyOptions, proceed, err
 	}
 	method := getEvent(req.Header)
-	if method != "ping" && method != "push" && method != "Push Hook" {
```

Comment:
The github doc links:  https://developer.github.com/v3/repos/hooks/ and https://developer.github.com/v3/activity/events/types/  ... the event name is just "push" (the latter link uses the heading of "PushEvent", but then when you read below the event name is just "push" .. i.e. that is what is in the header).

No hint of "Push Hook", which is clearly flagged in the gitlab doc:  https://docs.gitlab.com/ce/user/project/integrations/webhooks.html#push-events

Yeah, I think this has to be chalked up as an attempt to support gitlab, which won't work because the POST body format is too different and the JSON parsing will fail.